pkg/kubelet/rkt/rkt.go

```diff
@@ -2449,3 +2449,18 @@ func convertKubePortMappings(portMappings []kubecontainer.PortMapping) ([]appcty
 
 	return containerPorts, hostPorts
 }
+
+// makeCapabilities creates string slices from Capability slices
+func makeCapabilities(capAdd []v1.Capability, capDrop []v1.Capability) ([]string, []string) {
```

Comment:
looks like `MakeCapabilities` is in the docker tools version as well still.  Should it be left in the common code and removed from this impl and the docker impl?